### PR TITLE
LPS-91156 Add integration tests

### PR DESCRIPTION
**Pull request by marcelabc**

### CODE REVIEW

**Comment by marcelabc** [comment]
ci:test:sf

**Comment by marcelabc** [comment]
ci:test:relevant

File: modules/apps/data-engine/data-engine-service/src/main/java/com/liferay/data/engine/internal/executor/DEDataRecordCollectionListRecordRequestExecutor.java [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+import java.util.List;
+
+/**
+ * @author Marcela Cunha
+ */
+public class DEDataRecordCollectionListRecordRequestExecutor {
+
+	public DEDataRecordCollectionListRecordRequestExecutor(
+		DEDataEngineRequestExecutor deDataEngineRequestExecutor,
+		DDLRecordLocalService ddlRecordLocalService) {
```
Comment (jeyvison) [inline modules/apps/data-engine/data-engine-service/src/main/java/com/liferay/data/engine/internal/executor/DEDataRecordCollectionListRecordRequestExecutor.java]: Sort the parameters

```suggestion
		DDLRecordLocalService ddlRecordLocalService,
		DEDataEngineRequestExecutor deDataEngineRequestExecutor) {
```

File: modules/apps/data-engine/data-engine-service/src/main/java/com/liferay/data/engine/internal/executor/DEDataRecordCollectionListRecordRequestExecutor.java [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+
+		_deDataEngineRequestExecutor = deDataEngineRequestExecutor;
+		_ddlRecordLocalService = ddlRecordLocalService;
```
Comment (jeyvison) [inline modules/apps/data-engine/data-engine-service/src/main/java/com/liferay/data/engine/internal/executor/DEDataRecordCollectionListRecordRequestExecutor.java]: sort the assignments 

```suggestion
		_ddlRecordLocalService = ddlRecordLocalService;
		_deDataEngineRequestExecutor = deDataEngineRequestExecutor;
```

**Pull request closed without merging**